### ISSUE TITLE
GNS3VM Virtualbox doesn't have gns3server installed

### ISSUE THREAD

**Issue by LespiletteMaxime** [GNS3/gns3-gui#3298]
Hello,

I upgraded my GNS3VM from 2.2.29 to 2.2.30 and I get this error :

![image](https://user-images.githubusercontent.com/2920479/155718701-03df05ce-f944-4063-8a19-d7bc6f645853.png)

I though it was an issue with my GNS3VM so I tried with a fresh one and I get the same issue.

Here is the log from gns3server when it's started 
```
Traceback (most recent call last):
  File "/usr/local/bin/gns3server", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/gns3server/main.py", line 82, in main
    from gns3server.run import run
  File "/usr/local/lib/python3.8/dist-packages/gns3server/run.py", line 32, in <module>
    from gns3server.web.web_server import WebServer
  File "/usr/local/lib/python3.8/dist-packages/gns3server/web/web_server.py", line 37, in <module>
    from .route import Route
  File "/usr/local/lib/python3.8/dist-packages/gns3server/web/route.py", line 28, in <module>
    from ..controller.controller_error import ControllerError
  File "/usr/local/lib/python3.8/dist-packages/gns3server/controller/__init__.py", line 27, in <module>
    from .project import Project
  File "/usr/local/lib/python3.8/dist-packages/gns3server/controller/project.py", line 35, in <module>
    from .snapshot import Snapshot
  File "/usr/local/lib/python3.8/dist-packages/gns3server/controller/snapshot.py", line 32, in <module>
    from .import_project import import_project
  File "/usr/local/lib/python3.8/dist-packages/gns3server/controller/import_project.py", line 29, in <module>
    from .topology import load_topology
  File "/usr/local/lib/python3.8/dist-packages/gns3server/controller/topology.py", line 31, in <module>
    from ..schemas.topology import TOPOLOGY_SCHEMA
  File "/usr/local/lib/python3.8/dist-packages/gns3server/schemas/topology.py", line 22, in <module>
    from gns3server.schemas.compute import COMPUTE_OBJECT_SCHEMA
ImportError: cannot import name 'COMPUTE_OBJECT_SCHEMA' from 'gns3server.schemas.compute' (/usr/local/lib/python3.8/dist-packages/gns3server/schemas/compute/__init__.py)
```

Thanks.
Max

**Comment by LespiletteMaxime** [comment]
Hello,

Issue has been fixed by 2.2.31

**Comment by grossmj** [comment]
Just released, thanks for the confirmation the issue has been fixed.
